net/JonGallant.Azure.Identity.Extensions/DefaultAzureServiceBusCredential.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Azure.Identity;
+using Microsoft.Azure.ServiceBus;
+using Microsoft.Azure.ServiceBus.Primitives;
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace JonGallant.Azure.Identity.Extensions
+{
+    public class DefaultAzureServiceBusCredential : ITokenProvider
+    {
+        private TimeSpan timeout;
+        private DefaultAzureCredential defaultAzureCredential;
+
+        public DefaultAzureServiceBusCredential() : this(null, null)
+        {
+        }
+
+        public DefaultAzureServiceBusCredential(TimeSpan timeout) : this(null, timeout)
```

Comment:
Any reason why you need this ctr overload with timeout? `GetTokenAsync()` method has the option to provide the timeout and looks like that's the one that will be used.

Reply:
Removed